quipucords/tests/api/scantask/test_scantask_get_facts.py

```diff
@@ -1,55 +1,46 @@
 """Test ScanTask.get_facts."""
 
+from functools import partial
+
 import pytest
 
-from api.models import (
-    InspectResult,
-    RawFact,
-    ScanJob,
-    ScanTask,
-)
-from tests.utils.facts import random_name, random_value
+from api.models import ScanTask
+from tests.factories import ScanTaskFactory
 
 
 @pytest.fixture
-def raw_facts_list():
+def raw_facts_list(faker):
     """Fixture representing a list of random facts."""
-    systems = []
-    for _ in range(50):
-        systems.append({random_name(): random_value() for _ in range(100)})
-    return systems
+    return [
+        {
+            "name": faker.slug(),
+            "boolean": faker.pybool(),
+            "int": faker.pyint(max_value=100),
```

Comment:
regarding https://github.com/quipucords/quipucords/pull/2600#discussion_r1569288521

```suggestion
            faker.slug(): faker.slug(),
            faker.slug(): faker.pybool(),
            faker.slug(): faker.pyint(max_value=100),
```

Reply:
Or do the type-looking keys have a special meaning that I've overlooked?

Reply:
aaaah
except for "name", they don't.

Reply:
"name" (or any other hardcoded string) is required for sorting later